components/profile-controller/controllers/profile_controller.go

```diff
@@ -588,3 +591,17 @@ func removeString(slice []string, s string) (result []string) {
 	}
 	return
 }
+
+func updateNamespaceLabels(ns *corev1.Namespace) bool {
+	updated := false
+	if ns.Labels == nil {
+		ns.Labels = make(map[string]string)
+	}
+	for k, v := range kubeflowNamespaceLabels {
+		if _, ok := ns.Labels[k]; !ok {
```

Comment:
How come you aren't overriding existing labels?

Reply:
done

Reply:
according to https://github.com/kubeflow/kubeflow/pull/4533#discussion_r353412069, I think we should better not overwrite existing labels